### DISCUSSION TITLE
Student Developer Pack approved but no benefit

### DISCUSSION THREAD

**Discussion by lakred** [community/community#113523] [GitHub Education]
### Hi everyone,

Hi everyone,
Hello,

I have recieved confirmation from the [relevant application page](https://education.github.com/discount_requests/application) that my application has been approved:
image

However, I have recieved no email from neither my primary, nor education email to state that I have recieved any benefits. Logging into my account shows no benefits whatsoever.
<img width="297" alt="Screenshot 2024-03-21 alle 10 08 16" src="https://github.com/community/community/assets/115311821/3335e038-f7ff-444d-a885-2cf98bf7c26e">

**Comment by SebastienXia** [comment]
## Hello, @lakred 

Just wait 5 to 7 days after being approved, you will receive an email when you have active benefits.

![approved_purple](https://github.com/community/community/assets/105004912/6fddf1d3-d3aa-499f-b6ad-827042891ef5)

`[close the thread and mark as answered if this helped you resolve your issue]`
